src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
```

don't use transition: all, write only neede properties
```suggestion
    transition: transform 300ms, color 300ms;
```